CoroutineTemplate/data/src/main/java/co/nimblehq/coroutine/data/service/providers/MoshiBuilderProvider.kt

```diff
@@ -1,14 +1,11 @@
 package co.nimblehq.coroutine.data.service.providers
 
 import com.squareup.moshi.Moshi
-import com.squareup.moshi.adapters.Rfc3339DateJsonAdapter
 import com.squareup.moshi.kotlin.reflect.KotlinJsonAdapterFactory
-import java.util.*
 
 object MoshiBuilderProvider {
 
     val moshiBuilder: Moshi.Builder
         get() = Moshi.Builder()
-            .add(Date::class.java, Rfc3339DateJsonAdapter())
```

Comment:
it's a utility that most of the time we face with the API Date response type - does this relate to the `Clean Code refactor`?

Reply:
@sleepylee It should have a comment to explain why we need to add it?
I thought it was unnecessary as I couldn't see its usage in the template 🤔

Reply:
I agree this might seem to be odd without the comment and the usage 🤔  but I think it's only hard to the new users to the template 🤔  `Rfc3339` is a standard way of returning Date in this format: `2019-10-12T07:20:50.52Z` from the API - we faced this most of the time so parsing this become a norm to us 🤔 .

https://medium.com/easyread/understanding-about-rfc-3339-for-datetime-formatting-in-software-engineering-940aa5d5f68a

Perhaps you can add a comment or question in a separate PR later instead of unilaterally removing in this unrelated PR?

Reply:
@sleepylee Yep, I think I can revert it and add the comment inside like `// Utility for the API Date response type`. 
What do you think?

Reply:
or you can make it even more specific: 
`// Parse the DateTime in this format: [2019-10-12T07:20:50.52Z]`